fs/fs.go

```diff
@@ -449,6 +486,63 @@ func (fs *filesystem) MountLocal(ctx context.Context, mountpoint string, labels
 	return nil
 }
 
+// getLayerDiffMap will return the map of a compressed layer shasum to its uncompressed shasum.
+// If VerifyLocalMounts is disabled, this will return nil with no error.
+func (fs *filesystem) getLayerDiffMap(ctx context.Context, labels map[string]string, remoteStore resolverStorage) (map[string]digest.Digest, error) {
```

Comment:
Should this use an `ArtifactFetcher` instead so that we can access the local store?